app/src/main/java/fr/free/nrw/commons/campaigns/CampaignView.java

```diff
@@ -98,8 +99,10 @@ public void init() {
             if (campaign != null) {
                 tvTitle.setText(campaign.getTitle());
                 tvDescription.setText(campaign.getDescription());
-                SimpleDateFormat inputDateFormat = new SimpleDateFormat("yyyy-MM-dd");
-                SimpleDateFormat outputDateFormat = new SimpleDateFormat("dd MMM");
+                SimpleDateFormat inputDateFormat = new SimpleDateFormat("yyyy-MM-dd",
```

Comment:
Try using `dateInLocaleFormat` instead

Reply:
@maskaravivek I am not entirely sure how to use `dateInLocaleFormat` here. The method `dateInLocaleFormat` uses the pattern `dd MMM yyyy` and requires a Date parameter.

Reply:
@maskaravivek tried digging up the function, not sure how to use it in this case

Reply:
IMO we can start using the same format for campaigns too. 

@ashishkumar468 Can you confirm if there's a specific reason for using this format or can @RonakLakhotia go ahead and change it?

Reply:
@maskaravivek by same format do you mean using `dd MMM yyyy` as in the `dateInLocaleFormat` method?

Reply:
Yes.

Reply:
The `dateInLocaleFormat` takes in a Date object parameter, so I am not sure how to use that method in this case.

Reply:
@maskaravivek here is the function definition
`public static String dateInLocaleFormat(Date date)`

Reply:
Please undo this as well. Let the lint keep warning about this issue so that a proper fix can be taken up later.